designs/virtualization/4.6/Next-run-configuration/Next-run-configuration.md

```diff
@@ -0,0 +1,72 @@
+---
+parent: Virtualization
+version: 4.6
+---
+# Next run configuration
+
+As a user makes changes to a running Virtual machine they will need to wait for the VM to restart to see the changes applied (the changes won’t be applied until the next start of the VM).
+
+## Editing
+
+When a user clicks to edit/ update one of the attributes (f.e. Flavor) it will open a modal with an alert notification letting the user know that the VM must be restarted in order to see the changes applied. The alert will persist until they restart the VM.
+This alert will not appear until the user has made changes, so the user will then know that they will need to restart.
+
+In general, users will be able to clear the changes and keep what they had, but at the very first time the modal is opened in case the user hasn't saved any changes yet, the 'Clear changes' button will not be shown.
+
+To avoid confusion users will be able to view the current settings in The VM Details page.
+
+![user is editing values](img/EditOriginal.png)
+
+At this point the user can either just save the requested changes and postpone the restart or save and restart the VM.
+
+## Restart modal
+
+If the user chooses to save and restart the VM they will be asked again to ensure that and will get into a modal including an inline notification that restarting may be disruptive to some logged in users and a notification that the new setting will be applied upon restart. The notification will include a confirmation checkbox so they would be aware of the effect that the restart action might cause and a link to the logged in users list at the bottom of the VM Details page.
+
```

Comment:
@yfrimanm @matthewcarleton What happens when the user click `no` when asking him the second time about the restart?
do we save the changes and return to the main view or we return to the first modal ?

Reply:
@irosenzw TMU we will save the changes and return to the VM details page.